src/components/controls/controlHeader.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { HeaderContainer } from "./styles";
+import { AnnotatedTitle } from "./annotatedTitle";
+
+type Props = {
+  /** Title to display for the control. */
+  title: string
+
+  /** Informational tooltip element to display on hover. */
+  tooltip?: JSX.Element
+
+  /** Indicates mobile display. */
+  mobile: boolean
+}
+
+/**
+ * A header used by all non-panel controls, containing an informative title.
+ */
+export const ControlHeader = ({title, tooltip=undefined, mobile }: Props) => {
```

Comment:
This is because [exactOptionalPropertyTypes](https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes) in tsconfig is enabled. I don't think it's necessary, so I've pushed a commit to disable it: 67bffee2b0dd80066b32eba028d5b0b6b701d7ec